### PR TITLE
Added the Bash book to the book section

### PR DESCRIPTION
**Pull request by bharath-123**
Added the Bash book to book section. This book was in the online course section.

### CODE REVIEW

File: free-programming-books.md [hunk 1]
```diff
@@ -837,6 +837,7 @@
 * [The Bash Academy](http://www.bash.academy)
 * [The Linux Command Line](http://linuxcommand.org/tlcl.php) - William E. Shotts, Jr. (PDF)
 * [Writing Shell Scripts](http://linuxcommand.org/lc3_writing_shell_scripts.php) - William E. Shotts, Jr.
+* [Bash tutorial](http://gdrcorelec.ups-tlse.fr/files/bash.pdf) (PDF)
```
Comment (eshellman) [inline free-programming-books.md]: check the [guidlines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#guidelines) for help on passing tests. (alphabetization, include author.)

**Comment by eshellman** [comment]
I agree this is more of a book than a course.

**Comment by charlottetan** [comment]
this PR is for #2482

**Comment by charlottetan** [comment]
Looks like this PR has been abandoned? Might be worth closing out

**Pull request closed without merging**